apps/studio/src/components/PageEditor/constants.ts

```diff
@@ -10,12 +10,6 @@ export const DEFAULT_BLOCKS: Record<
     content: [
       {
         type: "paragraph",
-        content: [
-          {
-            type: "text",
-            text: "",
-          },
-        ],
```

Comment:
deletion required due to tiptap removing this property when the content of the paragraph is visually empty.